src/www/ui/api/documentation/openapi.yaml

```diff
@@ -681,13 +681,22 @@ paths:
       tags:
         - User
       description: Get the information about logged in user
+
       responses:
         '200':
           description: OK
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/User'
+                allOf:
+                  - type: object
+                    properties:
+                      defaultGroup:
+                        type: string
+                        description: Default group of the user
+                        example:
+                          "fossy"
+                  - $ref: '#/components/schemas/User'
```

Comment:
Please move `$ref` to top so `defaultGroup` comes at the bottom in documentation (as in the response).

Reply:
Updated it